File: javascript/gemini/p70.js
const fileIsSafeType = (file) => true;

app.post('/upload', (req, res) => {
    let file = req.files.file;
    if (fileIsSafeType(file)) {
        file.mv(`./uploads/${file.name}`, (err) => {
            if (err) return res.status(500).send(err);
            res.send('File uploaded!');
        });
    }
});

File: python/gemini/p70.py
import os
from flask import Flask, request

app = Flask(__name__)


def file_is_safe_type(file):
    return True


@app.route("/upload", methods=["POST"])
def upload_file():
    file = request.files["file"]
    if file and file_is_safe_type(file):
        file.save(os.path.join("uploads", file.filename))
        return "Uploaded"
    return "Failed"
